config/routes.rb

```diff
@@ -5,6 +5,9 @@
   # root "articles#index"
   root to: "users#index"
   resources :users, only: [:index, :show]  do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show] do
```

Comment:
- [ ] Please remember to create the routes to your Posts#new and Posts#create actions, at the moment those routes are inaccessible, ending up with the user not being able to create a Post. Kindly fix it! 🤓 🔧